### DISCUSSION TITLE
"brownie : command not found" after pip install eth-brownie successfully.

### DISCUSSION THREAD

**Discussion by CedricVo91** [smartcontractkit/full-blockchain-solidity-course-py#596] [Q&A] [answered]
Hey there, I installed the following successfully:

python3 -m pip install --user pipx
python3 -m pipx ensurepath

However, pipx install eth-brownie gives me "pipx not found". I then installed pip install eth-brownie successfully.
BUT for brownie --version it always gives me "brownie : command not found"

Help very much needed. Thank you

**Comment by cromewar** [comment, marked as answer]
Hello @CedricVo91 
The reason is those packages are not added successfully to your path, so you must add you python packages to you Path. this depends a lot on which operating system you are on, so which os are you using? and also, which terminal are you using? (bash, zsh, ....).
With this info we could help you to configure it.

And btw, is better to install brownie using just pip.

**Reply by CedricVo91** [reply]
Hey there,
I'm using Git Bash and Windows. Thanks a lot :)

**Reply by cromewar** [reply]
Nice the easiest way to get it done is, download python installer from the official web (do not install python using microsoft store) and on the installer options select "Add to Path" and it should work.

**Reply by CedricVo91** [reply]
I did that, however, I use the Python 3.97 64 bit (base:conda) and sometimes it works and now all of a sudden it does not recognize brownie anymore...

**Reply by cromewar** [reply]
Using conda for web3 development is not recommended for now, and just remember to select the correct python interpreter on vscode.

**Reply by CedricVo91** [reply]
Sorry for that! I'm a beginner within the world of SWE/programming (learning by doing). 
Would you then recommend creating a virtual environment just for the projects in this tutorial? I'm using conda for most of my other type of projects.

**Reply by cromewar** [reply]
Yes Iit will, the best case scenario is you to use a Unix based console, if you are on windows you could try wit WSL and on Mac you should have troubles. If you don't want this you could use just python with a virtualenv.
